### DISCUSSION TITLE
Have to login twice

### DISCUSSION THREAD

**Discussion by jmcbade** [kanboard/kanboard#5205] [Installation & Configuration]
### Checklist

- [x] I verified that Kanboard is correctly installed
- [x] I checked that the problem does not come from a plugin
- [x] I checked that the problem is not already reported

### Actual behaviour
This is a clean install and have duplicated on both Chrome and Safari.  Using MAC OS Big Sur 11.5.1. Host is using a MySql DB (not SqlLite).  kanboard-1.2.20

Login, logout, re-enter the user, pwd, login and get a "red fields" error as though nothing was entered.  Enter the User and Password AGAIN and login is successful.

### Expected behaviour

Expected the valid User and Password to work the first time.
### Steps to reproduce
Logout, from the login screen, re-enter User and Password.  Fails with red field as though nothing was entered..  Enter User and Password again, success.  Refreshing the screen first did not solve.

### Screenshots
![image](https://user-images.githubusercontent.com/1731707/129139476-e3c387e0-be40-4fa8-a9ac-130c711b6c49.png)

### Configuration

- Kanboard version: 
- Database type and version:
- PHP version:
- OS:
- Browser:

**Comment by jmcbade** [comment]
This is a clean install with admin and one user - updated passwords.

**Comment by rautamiekka** [comment]
Fill the configuration section and show the logs.
